core-common/src/main/java/org/zalando/nakadi/domain/EventTypeBase.java

```diff
@@ -67,11 +72,15 @@
 
     @Valid
     @Nullable
-    private ResourceAnnotations annotations;
+    private Map<
+            @AnnotationKey String,
```

Comment:
Maybe `ValidAnnotationKey` for a name?

Reply:
Looking at other annotation values: 
`Size`, `Max`, `Pattern`, `NotEmpty`, `Email`, I would say no...

Reply:
Right.  It's good like that.